src/Map/src/Icon/Icon.php

```diff
@@ -61,6 +61,7 @@ protected function __construct(
         protected IconType $type,
         protected int $width = 24,
         protected int $height = 24,
+        protected ?string $color = null, // Nouvelle propriété pour la couleur
```

Comment:
The comment does not bring any value

```suggestion
        protected ?string $color = null,
```